kythe/go/extractors/gcp/config/yamlgen/yamlgen.go

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Binary yamlgen generates a Cloud Build yaml config file for extracting a
+// repo using Kythe.
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+
+	"kythe.io/kythe/go/extractors/gcp/config"
+)
+
+var (
+	input  = flag.String("input", "", "The input config proto to read.")
+	output = flag.String("output", "cloudbuild.yaml", "The output yaml file to write.")
+)
+
+func main() {
+	flag.Parse()
+
+	yamlData, err := config.KytheToYAML(*input)
+	if err != nil {
+		log.Fatalf("failure converting %s to %s: %v", *input, *output, err)
+	}
+
+	if err := ioutil.WriteFile(*output, yamlData, 0644); err != nil {
```

Comment:
Does this take empty or "-" as meaning stdout? and similarly for the call to KytheToYAML?

Reply:
Re: writing to stdout. There's no magic pathname, but you could write to /dev/stdout probably.

Reply:
`/dev/stdout` does work, but "-" has to be handled specially.  Some of this is handled in our vfs package: https://github.com/kythe/kythe/blob/master/kythe/go/platform/vfs/vfs.go#L133